src/components/basic/form/Legend.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
```

Comment:
instead of creating a legend component, i would put this directly in the fieldset component. and then make the prop type changes as we talked about on chat.

Reply:
Working on removing Legend component and integrating the <legend> tag directly in to the FieldSet. Will commit afterwards.

Reply:
Updated and committed.